README.md

```diff
@@ -24,7 +24,7 @@ you use this method, the bridge can be run via `node app.js`.
 
 
 ### Requirements
- - Node.js v4 or above.
+ - Node.js v4.5 or above.
```

Comment:
I don't know, you committed it :stuck_out_tongue: https://github.com/matrix-org/matrix-appservice-irc/pull/290/commits/340b8b921b61328aa8075288fd9e2bbb203ddeb2